### ISSUE TITLE
portable: document requirements for minimal images

### ISSUE THREAD

**Issue by Prototik** [systemd/systemd#11870]
Currently unclear what exactly portable image should contain to properly work with portable services.  
With god blessing I found that following minimal working tree:
```
/
├── bin -> usr/bin
├── dev
├── etc
│   ├── machine-id
│   ├── os-release
│   └── resolv.conf
├── lib -> usr/lib
├── proc
├── run
├── sys
├── tmp
├── usr
│   ├── bin
│   │   └── // unit binaries here
│   └── lib
│       └── systemd
│           └── system
│               └── // units here
└── var
    └── tmp
```

So, almost all directories binded by systemd (by using Protect* Private* and others) to something appropriate, but it unable to do it if these directories doesn't exists in read-only image (squashfs).

On the other hand, maybe systemd shouldn't try to bind directories which not required (not present) by the image? Then images could be even lesser - just binaries, units, resolv.conf & machine-id & os-release

**Comment by poettering** [comment]
Fix waiting in #11989

**Comment by boucman** [comment]
Nice, thx.
